ipaserver/plugins/batch.py

```diff
@@ -77,14 +76,10 @@ class batch(Command):
         ),
     )
 
-    take_options = (
-        Str('version',
-            cli_name='version',
-            doc=_('Client version. Used to determine if server will accept request.'),
-            exclude='webui',
-            flags=['no_option', 'no_output'],
-            default=API_VERSION,
-            autofill=True,
```

Comment:
Shouldn't we set `deprecated=True` before removing the parameter?

Also, it is still available in `API.txt`.

Reply:
We don't need to because this tuple (`take_options`) is not something that is used anywhere. It was simply ignored since the very introduction of this command in 2011.

Now, the `version` option still exists because it is being injected by `Command.get_options()` unconditionally.

Reply:
The correct name of the tuple should have been `takes_options`, not `take_option`.
```
$ git grep takes_options|wc -l
1987
$ git grep take_options|wc -l
0
```

Reply:
Thank you, got it now.